beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/model/HousekeepingMetadata.java

```diff
@@ -126,6 +141,10 @@ public void setTableName(String tableName) {
     this.tableName = tableName;
   }
 
+  public String getPartitionName() {
```

Comment:
Can you add a line fo javadoc explaining the format of this name?

Reply:
@vedantchokshi ^

Reply:
???

Reply:
Looks good, also matches how Hive seem to describe this in the (very limited) javadoc on https://hive.apache.org/javadocs/r2.1.1/api/org/apache/hadoop/hive/metastore/IMetaStoreClient.html#partitionNameToVals-java.lang.String-